app/views/user_sessions/_greeting.html.erb

```diff
@@ -28,7 +28,7 @@
             <li><%= span_if_current ts('My History', key: 'header'), user_readings_path(current_user) %></li>
           <% end %>
           <li><%= span_if_current ts('My Preferences', key: 'header'), user_preferences_path(current_user) %></li>
-          <% if permit?('opendoors') %>
+          <% if permit?('opendoors', :get_user_method => 'current_user') %>
```

Comment:
Hound doesn't run on .html.erb files, so it didn't mention that you should probably change to the `get_user_method: 'current_user'` syntax here as well.